app/src/main/java/io/islnd/android/islnd/app/database/IslndDbHelper.java

```diff
@@ -85,6 +85,13 @@ public void onCreate(SQLiteDatabase db) {
 
                 " UNIQUE (" + IslndContract.ProfileEntry.COLUMN_USER_ID + ") ON CONFLICT REPLACE);";
 
+        final String SQL_CREATE_NOTIFICATION_TABLE = "CREATE TABLE " + IslndContract.NotificationEntry.TABLE_NAME + " (" +
+                IslndContract.NotificationEntry._ID + " INTEGER PRIMARY KEY," +
+                IslndContract.NotificationEntry.COLUMN_NOTIFICATION_USER_ID + " INTEGER NOT NULL," +
+                IslndContract.NotificationEntry.COLUMN_NOTIFICATION_TYPE + " INTEGER NOT NULL," +
+                IslndContract.NotificationEntry.COLUMN_POST_ID + " TEXT NOT NULL, " +
```

Comment:
If userId and postId reference a specific post, we might want this table to have a foreign key relation to the post table. It would be similar to the comment table.